deb/debian/rules

```diff
@@ -5,10 +5,10 @@ ARCH := $(shell uname -m)
 DISTRO=$(shell lsb_release -a 2>/dev/null |grep Codename |awk {'printf $$2;'})
 debian_version := $(shell dpkg-parsechangelog | sed -n 's/^Version: //p')
 
-ifeq ($(DISTRO),stretch)
+ifneq (,$(filter $(DISTRO),buster stretch))
 HIREDIS=libhiredis0.13
 MYSQL=libmariadbclient18
-else ifeq ($(DISTRO),xenial)
+else ifneq (,$(filter $(DISTRO),xenial bionic))
```

Comment:
Though bionic is based on buster, I think it should be a standalone change as the goal of this change is to include buster only. You can follow up change on this line right away. What do you think?

Reply:
Bionic is not really based on Buster — they're actually independent.
This is a minor drive-by change that fixes a harmless warning in the Bionic build (a wrong dependency exclusion flag that I noticed while editing this file), which doesn't merit a separate PR, IMO... I could also leave it as-is, too, but might as well do it here.
Happy to add it to the PR description, too.

Reply:
Got it, thanks for clarification and correct me!

Yes, please add the change into commit description when you merge the code back to master.